packages/babel-plugin-transform-es2015-classes/src/index.js

```diff
@@ -2,6 +2,8 @@ import LooseTransformer from "./loose";
 import VanillaTransformer from "./vanilla";
 import nameFunction from "babel-helper-function-name";
 
+const PURE_ANNOTATION = "#__PURE__";
```

Comment:
We could use this across multiple plugins? We might want to add it in a helper.

Reply:
@xtuc sure thing, whats the best place to add such?

Reply:
Too late but I had something like `babel-help-emit-pure-annotation` in mind but I'm not sure it will be used apart from here.

Reply:
I would just make it when we need/get to it. It's not a breaking change to swap it out either